### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by lionkor**
I saw a bunch of wrong and outdated code here and fixed them

### CODE REVIEW

File: bans.txt
Comment (Starystars67) [inline bans.txt:1]: These were examples to assist users, that is why they were here :)

**Review by Starystars67 — COMMENTED** [review]
Otherwise all good, thank you for updating!

File: bans.txt
Comment (lionkor) [reply bans.txt:1]: yes, but they're the wrong format. beammp ids are numbers, not beammp:number format.

**Comment by lionkor** [comment]
I've added two example beammp ids to bans.txt - they are both real users that are indefinitely suspended on the forum, so no harm done using them here :)

**Pull request merged**